opennav_docking_bt/CMakeLists.txt

```diff
@@ -16,7 +16,10 @@ find_package(behaviortree_cpp REQUIRED)
 
 # potentially replace with nav2_common, nav2_package()
 set(CMAKE_CXX_STANDARD 17)
-add_compile_options(-Wall -Wextra -Wpedantic -Werror -Wdeprecated -fPIC -Wshadow -Wnull-dereference)
+if(CMAKE_COMPILER_IS_GNUCXX OR CMAKE_CXX_COMPILER_ID MATCHES "Clang")
+  add_compile_options(-Wall -Wextra -Wpedantic -Werror -Wdeprecated -fPIC -Wshadow -Wnull-dereference)
+endif()
+set(CMAKE_WINDOWS_EXPORT_ALL_SYMBOLS ON)
```

Comment:
Can you check if its a windows OS to set this on?

Reply:
Done, thanks!